recipes/mender-client/debian-master/rules

```diff
@@ -1,42 +1,24 @@
 #!/usr/bin/make -f
+#
+#DH_VERBOSE=1
+# see FEATURE AREAS in dpkg-buildflags(1)
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+# see ENVIRONMENT in dpkg-buildflags(1)
+# package maintainers to append CFLAGS
+#export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+# package maintainers to append LDFLAGS
+#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
 %:
 	dh $@
 
-override_dh_auto_build:
-	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC="${CC}" \
-	GOOS=linux CGO_ENABLED=1 build
-
-install_targets_exclude=install-systemd install-modules-gen
-install_targets_all=$(shell grep -E '^install\-.*:' Makefile | cut -d: -f1)
-install_targets=$(filter-out $(install_targets_exclude),$(install_targets_all))
-override_dh_auto_install:
-	DESTDIR=debian/mender-client make $(install_targets)
-
-	# Install scripts/version file
-	install -d -m 755 debian/mender-client/etc/mender/scripts/
-	echo -n "3" > debian/mender-client/etc/mender/scripts/version
-
-	# MEN-6060: Make identity and inventory scripts into conf files so that
-	# they can be modified without risking that they get lost on an
-	# upgrade. To follow Debian policy, we must then put them in /etc and
-	# symlink to them, we can not simply mark them as conffiles:
-	# https://www.debian.org/doc/debian-policy/ch-files.html#location
```

Comment:
Shouldn't this be kept?

Reply:
Mby, dunno

Reply:
Yes, we should keep it. We can either keep this hook here or modify the install targets in our CMake to install the files in `etc` and create the symlinks from `/usr/share`

Reply:
Leaving this for now

Reply:
fixed